### ISSUE TITLE
Order Manager returns 200 instead of 404 on empty result list

### ISSUE THREAD

**Issue by tangobravo62** [dlr-eoc/prosEO#37]
When showing the list of orders for an empty database, an empty list is returned:
```
prosEO> order show
--- []

prosEO>
```
In line with other services, an error message should be returned, based on a 404 NOT_FOUND return code from the Order Manager service; compare to:
```
prosEO> processor class show
(E2970) No processor classes found for given search criteria
prosEO>
```

**Comment by tangobravo62** [comment]
The same applies for the Facility Manager.

**Comment by tangobravo62** [comment]
Implemented with commit c56850a
